pages.es/common/nxc-ssh.md

```diff
@@ -0,0 +1,25 @@
+# nxc ssh
+
+> Prueba y explota servidores SSH.
+> Vea también: `hydra`.
+> Más información: <https://www.netexec.wiki/ssh-protocol>.
+
+- Pulveriza la contraseña especificada contra una lista de nombres de usuario en el objetivo especificado:
+
+`nxc ssh {{192.168.178.2}} -u {{ruta/a/nombres_de_usuario.txt}} -p {{contraseña}}`
```

Comment:
```suggestion
`nxc ssh {{192.168.178.2}} -u {{ruta/al/nombres_de_usuario.txt}} -p {{contraseña}}`
```

Reply:
that suggestion is not correct in spanish language!